### PR TITLE
Fix items + Add Item Requirements

### PR DESCRIPTION
**Pull request by ENORMOUZ**
Fixed typo

### CODE REVIEW

**Comment by ENORMOUZ** [comment]
still lf unban from moulberry bush :pray:

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ This repository is used in order to store the various JSON files used for the Mi
 In order to contribute to the item repo you should enable the item editor tools by editing your config, in your configNew.json (`.minecraft\config\notenoughupdates\configNew.json`) ensure the following values are set:
 
 ```json
-"enableItemEditing": true,
```
Comment (jani270) [inline README.md:30]: Already done in another pr

**Review by jani270 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ENORMOUZ) [reply README.md]: Ok

**Comment by ENORMOUZ** [comment]
Adding combat requiments now

**Comment by jani270** [comment]
> idk how to resolve conflicts lol

Your pull request cant get merged with conflicts

**Comment by ENORMOUZ** [comment]
> > idk how to resolve conflicts lol
> 
> Your pull request cant get merged with conflicts

how do i resolve them? it says too complicated to resolve in web browser but i cant find the button in github desktop

**Comment by jani270** [comment]
> > > idk how to resolve conflicts lol
> > 
> > 
> > Your pull request cant get merged with conflicts
> 
> how do i resolve them? it says too complicated to resolve in web browser but i cant find the button in github desktop

You have to update your master, then there is a button "Choose a branch to merge into ... " and then select master there

**Comment by ENORMOUZ** [comment]
I have no idea why it commited every file (?)

**Comment by ENORMOUZ** [comment]
Yep I probably merged wrong but idk how to change it so it merges right

File: items/SCATHA;4.json [hunk 2]
```diff
@@ -17,12 +17,12 @@
...
     "§6Wormhole",
-    "§7§7Gives a §a{2}% §7chance to mine 2",
+    "§7§7Gives a §a{2}% §7to mine 2",
     "§7adjacent stone or hard stone.",
     "",
     "§6Gemstone Power",
-    "§7§7Gain §a+{3}% §7more Gemstone",
-    "§7Powder.",
+    "§7§7Gain §a+{3}% §7more §dGemstone",
+    "§7Powder§7.",
```
Comment (jani270) [inline items/SCATHA;4.json:25]: That makes no sense

**Review by jani270 — CHANGES_REQUESTED** [review]

File: items/SCATHA;4.json (hunk 2 above)
Comment (ENORMOUZ) [reply items/SCATHA;4.json:25]: Cause Hypixel changed the pet's lore to now make the Gemstone Powder pink colour

Comment (jani270) [reply items/SCATHA;4.json:25]: > Cause Hypixel changed the pet's lore to now make the Gemstone Powder pink colour

Yes but you still do §7 at the start of the second line and then you do §7 at the end again.

Comment (ENORMOUZ) [reply items/SCATHA;4.json]: was a typo oops

**Review by jani270 — CHANGES_REQUESTED** [review]
The removal of the ExtraAttribute "yearObtained" makes no sense as the year is in the lore, please revert

**Comment by jani270** [comment]
> The removal of the ExtraAttribute "yearObtained" makes no sense as the year is in the lore, please revert

Setting the edition to -1 could be smth unexpected for other mods, please revert

**Comment by ENORMOUZ** [comment]
> > The removal of the ExtraAttribute "yearObtained" makes no sense as the year is in the lore, please revert
> 
> Setting the edition to -1 could be smth unexpected for other mods, please revert

Oh it was set to -1 in some other item (i THINK spooky pie?) anyways reverting

**Comment by ENORMOUZ** [comment]
> The removal of the ExtraAttribute "yearObtained" and "year" makes no sense as the year is in the lore, please revert

Which item btw

**Comment by jani270** [comment]
> > > The removal of the ExtraAttribute "yearObtained" and "year" makes no sense as the year is in the lore, please revert
> > 
> > 
> > Which item btw
> 
> Cakes

Also used on the great spook stuff

**Review by jani270 — CHANGES_REQUESTED** [review]
The Holy Dragon Armor changes are useless

**Comment by jani270** [comment]
> > The §f changes on a lot of files are useless
> 
> You also removed the requirements of the Bingo Accessories should also be reverted

Please revert the Blaze Rod Collection changes on the blaze armor

**Comment by ENORMOUZ** [comment]
> > The §f changes on a lot of files are useless
> 
> You also removed the requirements of the Bingo Accessories should also be reverted

They dont have requirements anymore atleast on ah

**Comment by ENORMOUZ** [comment]
> The §f changes on a lot of files are useless

Yeah its because items in auction house have &f&f added in front of their names, so if I click the item from the auction house into the item editor, it keeps them.

**Comment by jani270** [comment]
> Please revert the Blaze Rod Collection changes on the blaze armor

You forgot the Frozen Blaze Armor

**Comment by jani270** [comment]
> > > > The removal of the ExtraAttribute "yearObtained" and "year" makes no sense as the year is in the lore, please revert
> > > 
> > > 
> > > Which item btw
> > 
> > 
> > Cakes
> 
> Also used on the great spook stuff

Also revert the Edition changes on the great spook items

**Comment by ENORMOUZ** [comment]
> > > The §f changes on a lot of files are useless
> > 
> > 
> > Yeah its because items in auction house have &f&f added in front of their names, so if I click the item from the auction house into the item editor, it keeps them. Should the &f&f be removed from every item which has it?
> 
> Well you should remove it on items where you change nothing else

bor time to go through 4k files

**Comment by jani270** [comment]
> > > > The §f changes on a lot of files are useless
> > > 
> > > 
> > > Yeah its because items in auction house have &f&f added in front of their names, so if I click the item from the auction house into the item editor, it keeps them. Should the &f&f be removed from every item which has it?
> > 
> > 
> > Well you should remove it on items where you change nothing else
> 
> bor time to go through 4k files

You edited 886, no need to go through 4k files

**Comment by ENORMOUZ** [comment]
> > > > > The §f changes on a lot of files are useless
> > > > 
> > > > 
> > > > Yeah its because items in auction house have &f&f added in front of their names, so if I click the item from the auction house into the item editor, it keeps them. Should the &f&f be removed from every item which has it?
> > > 
> > > 
> > > Well you should remove it on items where you change nothing else
> > 
> > 
> > bor time to go through 4k files
> 
> You edited 886, no need to go through 4k files

Can I just remove it from every item that has it lol

**Comment by ENORMOUZ** [comment]
ok will do

**Comment by ENORMOUZ** [comment]
Should be fixed

File: items/AATROX_BATPHONE.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
-{
```
Comment (jani270) [inline items/AATROX_BATPHONE.json:1]: ???????

**Review by jani270 — CHANGES_REQUESTED** [review]

**Comment by jani270** [comment]
> > > The removal of the ExtraAttribute "yearObtained" makes no sense as the year is in the lore, please revert
> > 
> > 
> > Setting the edition to -1 could be smth unexpected for other mods, please revert
> 
> Oh it was set to -1 in some other item (i THINK spooky pie?) anyways reverting

Still set to -1, revert.

**Comment by jani270** [comment]
https://i.imgur.com/KzpuRqR.png ??????

**Comment by jani270** [comment]
https://i.imgur.com/z9ZVhnF.png ????

**Comment by jani270** [comment]
Also revert your changes to the maxed ench book

**Comment by jani270** [comment]
Im gonna be honest with you, if i find more shit im just gonna close it.

**Comment by ENORMOUZ** [comment]
oops will do soonTM

**Comment by jani270** [comment]
soonTM
